schemas/zowe-yaml-schema.json

```
@@ -796,6 +796,15 @@
         "applId": {
           "type": "string",
           "description": "Appl ID of your z/OSMF instance."
+        },
+        "authentication": {
+          "type": "object",
+          "properties": {
+            "scheme": {
+              "type": "string",
+              "description": "Authentication scheme for z/OSMF."
```

```suggestion
              "description": "Authentication scheme for z/OSMF.",
              "enum": ["httpBasicPassTicket", "zosmf"]
```

I guess we can validate the values itself. No other string makes sense.